pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/patch/DeletePatchTest.java

```diff
@@ -0,0 +1,154 @@
+package org.broadinstitute.dsm.patch;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.Gson;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.DbTxnBaseTest;
+import org.broadinstitute.dsm.db.OncHistoryDetail;
+import org.broadinstitute.dsm.db.SmId;
+import org.broadinstitute.dsm.db.Tissue;
+import org.broadinstitute.dsm.exception.DuplicateException;
+import org.broadinstitute.dsm.model.patch.BasePatch;
+import org.broadinstitute.dsm.model.patch.DeleteOncHistoryPatch;
+import org.broadinstitute.dsm.model.patch.DeletePatchFactory;
+import org.broadinstitute.dsm.model.patch.DeleteTissuePatch;
+import org.broadinstitute.dsm.model.patch.Patch;
+import org.broadinstitute.dsm.model.patch.PatchFactory;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+@Slf4j
+public class DeletePatchTest extends DbTxnBaseTest {
+    private static final String participantGuid = "TEST_PARTICIPANT";
+    private static String participantId;
+    private static String instanceName = "testInstance";
+    private static List<String> createdOncHistoryIds = new ArrayList<>();
+    private static List<String> createdTissueIds = new ArrayList<>();
+    private static List<String> createdSmIdPks = new ArrayList<>();
+    private static TestParticipantUtil testParticipantUtil = new TestParticipantUtil(instanceName, "testInstanceGuid", "", "test");
+
+    @BeforeClass
+    public static void createParticipantAndMedicalRecords() {
+        participantId = testParticipantUtil.createParticipantForStudy("TEST_PARTICIPANT");
+    }
+
+    @AfterClass
+    public static void removeEverything() {
+        testParticipantUtil.deleteSmId(createdSmIdPks);
+        testParticipantUtil.deleteTissue(createdTissueIds);
+        List<String> participantIds = new ArrayList<>();
+        participantIds.add(participantGuid);
+        testParticipantUtil.deleteOncHistoryDetailAndMedicalRecord(createdOncHistoryIds, participantIds);
+    }
+
+    @Test
+    public void testIsDeletePatchMethod() {
+        String oncHistoryDeleteJson =
```

Comment:
Another way to do this that might be more readable is to new up an instance of `Patch` code-wise, and skip all the escaped json to deserialization to get `oncHistoryDeletePatch` in the desired state.